lib/api/slack.ex

```diff
@@ -27,20 +28,19 @@ defmodule Aprb.Api.Slack do
         raise("Unauthorized")
       end
 
-      subscriber = case Repo.get_by(Subscriber, user_id: params[:user_id]) do
+      subscriber = case Repo.get_by(Subscriber, channel_id: params[:channel_id]) do
```

Comment:
I see. I was thinking that subscriber = channel, but I see where you were going with this. Still maybe it should be named for what it is. You can keep the user id for purposes such as "who subscribed this channel to this topic?" But it cannot be a key.